### ISSUE TITLE
Could not create input directory

### ISSUE THREAD

**Issue by richchurcher** [gobanos/cargo-aoc#9]
Hello again :smile: with the latest update there's a panic if the input directory already exists, it seems?

```sh
$ cargo aoc input
Requesting input for year 2018, day 2 ...
thread 'main' panicked at 'Could not create input directory: Os { code: 17, kind: AlreadyExists
, message: "File exists" }', libcore/result.rs:1009:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```
The contents of the directory were:
```sh
21:56 $ tree
.
└── 2018
    └── day1.txt

1 directory, 1 file
```
And as you can see, I was looking for day 2. Been fun using the tool so far though!

**Comment by gobanos** [comment]
Hi, nice to see you again !

This issue was corrected by @petreeftime in `cargo-aoc` 0.1.3, run `cargo install cargo-aoc --force` ;)

**Comment by richchurcher** [comment]
Argh, I can't keep up! Ty!
